iroh-net/src/endpoint.rs

```diff
@@ -294,6 +294,7 @@ impl StaticConfig {
 }
 
 /// Creates a [`quinn::ServerConfig`] with the given secret key and limits.
+// TODO: this return type can not longer be used anywhere in our public API.  Remove this.
 pub fn make_server_config(
```

Comment:
Ah. Good to know. I wanted move this function into `StaticConfig::create_server_config`!
I can do so now with good conscience.